src/Symfony/Component/Security/Http/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ CHANGELOG
  * Add support for closures in `#[IsGranted]`
  * Add `OAuth2TokenHandler` with OAuth2 Token Introspection support for `AccessTokenAuthenticator`
  * Add discovery support to `OidcTokenHandler` and `OidcUserInfoTokenHandler`
+ * Add ability for authenticators to explain why they support the request or not
```

Comment:
```suggestion
 * Add ability for authenticators to explain why they does not support the request
```
If they support it, no reason is set, right?

Reply:
s/does/do

Reply:
Updated, thanks